From 61efc7250aa52840d6ed812b2ea5851fa4fd9b27 Mon Sep 17 00:00:00 2001
From: Mauro Rossi <issor.oruam@gmail.com>
Date: Sun, 28 Feb 2021 20:19:21 +0100
Subject: [PATCH 093/100] ANDROID: binder: add support for RT prio inheritance.
 (v4)

(v4) Changelog:
- Use MAX_RT_PRIO instead of MAX_USER_RT_PRIO after commit
  ae18ad281e82 ("sched: Remove MAX_USER_RT_PRIO")
---
 drivers/android/binder.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/android/binder.c b/drivers/android/binder.c
index 171d438c573d..81eb0a142694 100644
--- a/drivers/android/binder.c
+++ b/drivers/android/binder.c
@@ -637,7 +637,7 @@ static int to_userspace_prio(int policy, int kernel_priority)
 	if (is_fair_policy(policy))
 		return PRIO_TO_NICE(kernel_priority);
 	else
-		return MAX_USER_RT_PRIO - 1 - kernel_priority;
+		return MAX_RT_PRIO - 1 - kernel_priority;
 }
 
 static int to_kernel_prio(int policy, int user_priority)
@@ -645,7 +645,7 @@ static int to_kernel_prio(int policy, int user_priority)
 	if (is_fair_policy(policy))
 		return NICE_TO_PRIO(user_priority);
 	else
-		return MAX_USER_RT_PRIO - 1 - user_priority;
+		return MAX_RT_PRIO - 1 - user_priority;
 }
 
 static void binder_do_set_priority(struct task_struct *task,
-- 
2.17.1

